internal/app/go-filecoin/node/node_test.go

```diff
@@ -157,8 +156,6 @@ func TestNodeStartMining(t *testing.T) {
 	seed.GiveMiner(t, minerNode, 0) // TODO: update to accommodate new go-fil-markets integration
 	// Start mining give error for fail to get miner actor from the heaviest tipset stateroot
 	assert.Contains(t, minerNode.StartMining(ctx).Error(), "failed to setup mining")
-	_, err := storage.NewMiner()
```

Comment:
Unskip this test and get it passing

Reply:
These tests depend on new storage miner actor functionality that isn't currently integrated (i.e. they currently fail because we fail to retrieve valid SectorInfo structs from the miner actor). I changed the skip string to note this.